contributors/guide/contributor-cheatsheet/README-uk.md

```diff
@@ -0,0 +1,322 @@
+- [Triage Dashboard] - Агрегування схожих помилок для їх кращого виправлення.
+- [Velodrome] - Дашборд для відстеження статусу виконання задач і тестів.
+
+### Важливі поштові адреси
+
+- community@kubernetes.io - Написати листа комусь із команди спільноти (SIG Contributor
+  Experience) щодо непорозумінь у спільноті.
+- conduct@kubernetes.io - Написати листа до Комітету з контролю поведінки в Kubernetes, приватна розсилка.
+- github@kubernetes.io - Надіслати приватного листа до [Адміністративної команди GitHub] з приводу дражливих питань.
+- steering@kubernetes.io - Надіслати листа до комітету управління. Загальнодоступна адреса із публічним архівом.
+- steering-private@kubernetes.io - Надіслати листа до комітету управління у приватному порядку, з приводу дражливих питань.
+- social@cncf.io - Зв’язатися із командою соціальних мереж CNCF; блог, Twitter акаунт, решта соціальних сервісів.
+
+### Інші корисні посилання
+
+- [Developer Statistics] - Переглянути статистику розробки за всіма проектами CNCF.
+- [Kubernetes Patch Release] Розклад патч-релізів Kubernetes і контактна інформація команди.
+
+---
+
+## Ефективне спілкування в GitHub
+
+### Як бути найкращими одне для одного
+
+Насамперед, ознайомтеся із [Кодексом поведінки Спільноти].
+
+#### Приклади хорошого/поганого спілкування
+
+Коли ви про щось питаєте чи шукаєте допомоги, будь ласка, будьте ввічливі:
+
+  🙂 “X не компілюється, коли я роблю Y. Є якісь припущення?”
+
+  😞 “X не працює! Будь ласка, виправіть це!”
+
+Якщо ви відхиляєте PR, будь ласка, надайте інформативне та доброзичливе пояснення, чому запропоновані зміни не можуть бути об'єднані з основною гілкою (merged).
+
+🙂 “Я відхиляю цей PR тому, що ця функціональність не може підтримувати сценарій використання X. У запропонованій формі її краще реалізувати із інструментом Y. Дякую за вашу роботу.”
+
+😞 “Чому це не узгоджується із домовленостями щодо API? Це має бути зроблене в іншому місці!”
+
+---
+
+## Внесок до проекту
+
+### Підписання CLA
+
+Перш ніж зробити свій внесок, ви повинні [підписати Ліцензійну угоду учасника (CLA)][cla]. Ваш внесок до проекту Kubernetes може бути прийнято _лише_ за умови, що ви чи ваша компанія підписали CLA.
+
+Якщо у вас виникли труднощі із підписанням CLA, дотримуйтесь рекомендацій [інструкції з вирішення проблем CLA].
+
+### Відкрити та відреагувати на Issue
+
+GitHub Issue є основним джерелом для відстеження інформації з таких питань, як звіти про помилки, запити на вдосконалення, або для повідомлень, наприклад, про невдалі тести. Вони **не** призначені для [підтримки користувачів]. Для цього скористайтеся
+[інструкцією із усунення несправностей], повідомте про проблему до [Stack Overflow] чи [Kubernetes форум].
+
+**Посилання:**
+- [Мітки]
+- [Команди Prow][commands]
+
+
+#### Створення Issue
+
+- Скористайтеся шаблоном issue, якщо такий є. Використання відповідного шаблона допоможе іншим учасникам, які відповідатимуть на ваш запит.
+  - Дотримуйтесь рекомендацій, наведених у самому шаблоні.
```

Comment:
Do we need an extra space at the beginning of the line here?

Reply:
As far as I can see the extra space is present in the initial formatting so I left it in my version as well. Please check it at https://github.com/anastyakulyk/community/edit/master/contributors/guide/contributor-cheatsheet/README.md

Reply:
@anastyakulyk sorry, my bad - it's a different indent type. Looks good to me, thanks!